static/js/scripts.js

```diff
@@ -2,4 +2,34 @@
 var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
 var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
   return new bootstrap.Tooltip(tooltipTriggerEl)
-})
\ No newline at end of file
+})
+
+// Add map to index.html page
+var map = L.map('map').setView([53.27, -6.20], 13);
```

Comment:
could you maybe wrap your code in a function as well?

Reply:
@stefdworschak I'm going to resolve this for now and we can continue to refactor and add some reusable functions in due course. 🤜